app/database/persisted_queries.js

```diff
@@ -0,0 +1,75 @@
+const { knex } = require("./index");
+const redis = require("../redis");
+const DEFAULT_TTL = 12 * 3600;
+
+module.exports.count = async () => {
+	return (await knex("persisted_queries").count("key", { as: "amount" }))[0]
+		.amount;
+};
+
+module.exports.list = async ({
+	searchFragment = "",
+	limit = 100,
+	offset = 0
+}) => {
+	const rows = await knex("persisted_queries")
+		.select(["query", "key", "added_time"])
+		.where("query", "like", `%${searchFragment}%`)
+		.offset(offset)
+		.limit(limit);
+
+	return rows;
```

Comment:
My IDE would want me to do this change :D
```suggestion
	return await knex("persisted_queries")
		.select(["query", "key", "added_time"])
		.where("query", "like", `%${searchFragment}%`)
		.offset(offset)
		.limit(limit);
```